File: Java/limbo1.java
import java.util.*;
import java.io.*;
import java.math.*;

public class limbo1 {
    public static void main(String[] args) throws Exception {
        BufferedReader sc = new BufferedReader(new InputStreamReader(System.in));
        BufferedWriter dc = new BufferedWriter(new OutputStreamWriter(System.out));

        BigInteger two = new BigInteger("2");

        int cases = Integer.parseInt(sc.readLine());
        while (cases-- > 0) {
            String[] nums = sc.readLine().split(" ");
            BigInteger left = new BigInteger(nums[0]);
            BigInteger right = new BigInteger(nums[1]);

            // gauss formulas
            // 1 + (left * (left + 1)) / 2
            // (right * (left + left + right + 3)) / 2

            BigInteger val1 = BigInteger.ONE.add(left.multiply(left.add(BigInteger.ONE)).divide(two));
            BigInteger val2 = right.multiply(left.add(left.add(right.add(BigInteger.ONE.add(two))))).divide(two);
            dc.write(val1.add(val2).toString() + "\n");
        }

        dc.close();
        sc.close();
    }
}

File: C++/limbo1.cpp
#include <bits/stdc++.h>

typedef unsigned long long ull;

using namespace std;

int main() {
    ios_base::sync_with_stdio(false);
    cin.tie(nullptr);
    ull cases, left, right;
    cin >> cases;
    while (cases-- > 0) {
        cin >> left >> right;
        // use gauss formula
        // 1 + (left * (left + 1)) / 2
        // (right * (left + left + right + 3)) / 2
        ull val1 = 1 + (left * (left + 1)) / 2;
        ull val2 = (right * (left + left + right + 3)) / 2;
        cout << val1 + val2 << "\n";
    }
    return 0;
}